.cargo/config.toml

```diff
@@ -12,6 +12,7 @@ rustflags = [
   "-Wclippy::pedantic",
   # Allowed pedantic lints
   "-Wclippy::char_lit_as_u8",
+  "-Aclippy::cast_precision_loss",
```

Comment:
I suggest moving this into a dedicated PR. I found the rule useful so far (I've been liberal with allowing the rule in methods where I knew it isn't a problem)